README.md

```diff
@@ -82,7 +82,7 @@ Rack::Attack.enabled = false
 b) For __rack__ applications:
 
 ```ruby
-# In config.ru
+# In config.rb
```

Comment:
Hi,

This is not a typo. Is a reference to the rack up file as in https://github.com/rack/rack#usage.

Thanks.